config.go

```diff
@@ -43,12 +44,20 @@ func parseSshConfigFileSection(content string) *SshConfigFileSection {
 
 // parseSshConfigFile parses the ~/.ssh/config file and build a list of section
 func parseSshConfigFile(path string) (map[string]*SshConfigFileSection, error) {
+
+	sections := make(map[string]*SshConfigFileSection)
+
+	if _, err := os.Stat(path); err != nil && os.IsNotExist(err) {
+		log.Debugf("cannot find ssh config file: %s", path)
+		return sections, nil
+	}
+
 	log.Debugf("parsing ssh config file: %s", path)
 	content, err := ioutil.ReadFile(path)
 	if err != nil {
```

Comment:
Instead of doing a stat above you can just handed the IsNotExist error here that will come from ReadFile.

```go
if err != nil {
    if os.IsNotExist(err) {
        return make(map[string]*SshConfigFileSection), nil
    }
    return nil, err
}
```